src/NServiceBus.AzureStoragePersistence/Timeout/TimeoutLogic/TimeoutPersister.cs

```diff
@@ -39,34 +57,36 @@ public TimeoutPersister(Configure config)
             var results = new List<Tuple<string, DateTime>>();
            
             var now = DateTime.UtcNow;
-            var context = new ServiceContext(account.CreateCloudTableClient()) {IgnoreResourceNotFoundException = true};
+
+
+            var timeoutDataTable = client.GetTableReference(timeoutDataTableName);
+            var timeoutManagerDataTable = client.GetTableReference(timeoutManagerDataTableName);
+
             TimeoutManagerDataEntity lastSuccessfulReadEntity;
-            var lastSuccessfulRead = TryGetLastSuccessfulRead(context, out lastSuccessfulReadEntity)
+            var lastSuccessfulRead = TryGetLastSuccessfulRead(timeoutManagerDataTable, out lastSuccessfulReadEntity)
                                             ? lastSuccessfulReadEntity.LastSuccessfullRead
                                             : default(DateTime?);
 
-            IOrderedEnumerable<TimeoutDataEntity> result;
             IQueryable<TimeoutDataEntity> query;
 
             if (lastSuccessfulRead.HasValue)
             {
-                query = from c in context.TimeoutData
-                            where c.PartitionKey.CompareTo(lastSuccessfulRead.Value.ToString(PartitionKeyScope)) >= 0
-                            && c.PartitionKey.CompareTo(now.ToString(PartitionKeyScope)) <= 0
-                                && c.OwningTimeoutManager == _endpointName
+                query = from c in timeoutDataTable.CreateQuery<TimeoutDataEntity>()
+                            where c.PartitionKey.CompareTo(lastSuccessfulRead.Value.ToString(partitionKeyScope)) >= 0
+                            && c.PartitionKey.CompareTo(now.ToString(partitionKeyScope)) <= 0
+                                && c.OwningTimeoutManager == endpointName
                         select c;
             }
             else
             {
-                query = from c in context.TimeoutData
-                        where c.OwningTimeoutManager == _endpointName
+                query = from c in timeoutDataTable.CreateQuery<TimeoutDataEntity>()
+                        where c.OwningTimeoutManager == endpointName
                         select c;
             }
 
-            result = query
-                        .Take(1000) // fixes isue #208. 
-                        .AsTableServiceQuery(context) // fixes issue #191
-                        .ToList().OrderBy(c => c.Time);
+            var result = query
+                .Take(1000) // fixes isue #208. 
```

Comment:
@SeanFeldman  remove `fixes issue`. either it is important enough to have a real comment or not.